### PR TITLE
💄 style: Update deepseek V3 model

### PR DESCRIPTION
**Pull request by PaperPlaneDeemo**
Update Deepseek V3 and description from deepseek official announcement. Pricing has not been updated yet; it needs to be updated in 45 days on February 8, 2025.

#### 💻 变更类型 | Change Type

- [ ] ✨ feat
- [ ] 🐛 fix
- [ ] ♻️ refactor
- [x] 💄 style
- [ ] 👷 build
- [ ] ⚡️ perf
- [ ] 📝 docs
- [ ] 🔨 chore

#### 🔀 变更说明 | Description of Change
https://api-docs.deepseek.com/zh-cn/news/news1226

### CODE REVIEW

**Review by PRCOMEDOWN — COMMENTED** [review]
The commit `4402a478a804914173876960581bd85d02f93418` made the following changes to the file `src/config/modelProviders/deepseek.ts`:

1. Updated the `displayName` from 'DeepSeek V2.5' to 'DeepSeek V3'.
2. Modified the `description` text to reflect the latest evaluation results and performance alignment of DeepSeek V3.
3. Changed the `releasedAt` date from '2024-09-05' to '2024-12-26'.

These changes update the information about the DeepSeek model to reflect its latest version and capabilities.

File: src/config/modelProviders/deepseek.ts
Comment (arvinxx) [inline src/config/modelProviders/deepseek.ts:1]: the pricing is updated too

Comment (PaperPlaneDeemo) [reply src/config/modelProviders/deepseek.ts:1]: The original price will remain until February 9, 2025, so do I need to update the price now?

Comment (arvinxx) [reply src/config/modelProviders/deepseek.ts:1]: yes. Unless you promise to update it at 2025/02/09, or this update will not change again for long time

Comment (PaperPlaneDeemo) [reply src/config/modelProviders/deepseek.ts:1]: Done

**Pull request merged**